tutorials/visualize-timeseries-data-timescaledb-grafana/index.mdx

```diff
@@ -5,63 +5,36 @@ meta:
 content:
   h1: How to visualize time-Series data with TimescaleDB and Grafana
   paragraph: This page shows how to visualize time-series data with TimescaleDB and Grafana
-steps:
-  - step: Creating a new database
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/visualize-timeseries-data-timescaledb-grafana/#creating-a-new-database
-  - step: Enabling the TimescaleDB extension
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/visualize-timeseries-data-timescaledb-grafana/#Enabling-the-TimescaleDB-extension
-  - step: Prepare the table for TimescaleDB
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/visualize-timeseries-data-timescaledb-grafana/#Prepare-the-table-for-TimescaleDB
-  - step: Adding and removing data
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/visualize-timeseries-data-timescaledb-grafana/#Adding-and-removing-data
-  - step: Querying data
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/visualize-timeseries-data-timescaledb-grafana/#Querying-data
-  - step: Visualizing data with Grafana
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/tutorials/visualize-timeseries-data-timescaledb-grafana/#Visualizing-data-with-Grafana
 categories:
   - storage
 tags: time-Series data TimescaleDB Grafana Database
 dates:
-  validation: 2021-05-10
+  validation: 2022-03-16
   posted: 2019-12-10
 ---
 
-## TimescaleDB - Overview
-
-[TimescaleDB](https://www.timescale.com/) is a time series database built on top of [PostgreSQL](/tutorials/install-postgresql/).  
-A time-series database typically stores “time-series data” as the name indicates. It can be seen as a sequence of data points, that measure the same thing over time and store the measurement results in time order.  
+[TimescaleDB](https://www.timescale.com/) is a time series database built on [PostgreSQL](/tutorials/install-postgresql/).  
+A time-series database typically stores “time-series data”, as the name indicates. It can be seen as a sequence of data points, that measure the same thing over time and store the measurement results in time order.  
 Each of these measurements is paired with a timestamp, defined by a name and a set of labeled dimensions (or “tags”).
 
-Time-series data is everywhere around us and an important point of our everyday life. Whether it is a factory that measures the production output of a specific machine, a farmer observing the humidity of the soil or a city measuring the regularity between trains.
+Time-series data is everywhere around us and an important part of our everyday life. Whether it is a factory that measures the production output of a specific machine, a farmer observing the humidity of the soil or a city measuring the regularity between trains.
```

Comment:
```suggestion
Time-series data is everywhere around us and an important part of our everyday life. Whether it is a factory that measures the production output of a specific machine, a farmer observing the humidity of the soil or a city measuring the regularity between trains, all of this is time-series data.
```

Reply:
I think gramatically either it needs something at the end, or it needs to start with something other than "whether"